frontend/src/components/ChangeProfileForm/ChangeProfileForm.js

```diff
@@ -50,17 +50,19 @@ export default function ChangeProfileForm({
       setShowError({ ...showError, errorEmail: true });
       return;
     }
-    if (!PASSWORD_REGEX.test(password)) {
-      setShowError({ ...showError, errorPassword: true });
-      return;
-    }
-    if (!PASSWORD_REGEX.test(newPassword)) {
-      setShowError({ ...showError, errorNewPasswordRegex: true });
-      return;
-    }
-    if (newPassword === password) {
-      setShowError({ ...showError, errorNewPasswordMatch: true });
-      return;
+    if (password !== '' && newPassword !== '') {
```

Comment:
Is this necessary?

Reply:
If both of them are not empty then we check if they are regex valid. If they are both empty, we ignore the regexes

Reply:
But doesn't regex do the same thing?

Reply:
No, I don't think so